plugins/org.jboss.tools.forge.runtime/META-INF/MANIFEST.MF

```diff
@@ -37,6 +37,7 @@ Export-Package: bootpath,
    org.jboss.forge.addon.projects,
    org.jboss.forge.addon.dependencies,
    org.jboss.forge.addon.resource",
+ org.jboss.forge.addon.projects.stacks,

```

Comment:
If I don't declare it, I get a ClassNotFoundException. Probably because org.jboss.forge.addon.projects.Project has a Stack getStack() method.